layouts/webinars/single.html

```diff
@@ -39,6 +39,9 @@ <h2>{{ .title }}</h2>
                             <pulumi-datetime class="uppercase text-orange text-sm font-bold" date="{{ .sortable_date }}"></pulumi-datetime>
                         {{ end }}
                         <p class="uppercase text-orange text-sm font-bold mt-2">DURATION: {{ .duration }}</p>
+                        {{ if .youtube_url }}
+                            <a href="{{ .youtube_url }}" target="_blank" rel="noopener noreferrer" class="btn">WATCH ON DEMAND</a>
```

Comment:
Super minor, but might suggest WATCH NOW as a CTA if it's just going to take me where when I click it.

Reply:
👍